tests/integration/test_sts.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from localstack.utils.aws import aws_stack
+
+
+class TestSTSIntegrations(unittest.TestCase):
+    def setUp(self):
+        self.sts_client = aws_stack.connect_to_service('sts')
+
+    def test_assume_role(self):
+        test_role_session_name = 's3-access-example'
+        test_role_arn = 'arn:aws:sts::123456789012:role/rd_role'
```

Comment:
Can we please change this to:
```
        test_role_arn = 'arn:aws:sts::000000000000:role/rd_role'
```
From the external user's perspective, we should always be using the default account ID `000000000000` in LocalStack (which is also configurable). Mocked/hard-coded account IDs like `123456789012` (or other values for other APIs) should remain an internal implementation detail - hence we're performing the conversion between these formats in the API proxy listener.